TodoList/View Controllers/TodoListViewController.swift

```diff
@@ -55,15 +55,15 @@ class TodoListViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
         
         if editingStyle == .delete {
-            todos.remove(at: indexPath.row)
+            todos.items.remove(at: indexPath.row)
             tableView.deleteRows(at: [indexPath], with: .automatic)
         }
     }
     
     // MARK: - TableView Delegate
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         // Toggle item completion state
-        todos[indexPath.row].isComplete = !todos[indexPath.row].isComplete
+        todos.items[indexPath.row].isComplete = !todos.items[indexPath.row].isComplete
```

Comment:
I know this is not you and it's also a nitpick/suggestion, but I really dislike this line of code. I rather replacing the struct in the array:
```
let item = todos.items[indexPath.row]
item.isComplete = !item.isComplete
todos.items[indexPath.row] = item
``` 

What do you think?

Reply:
Much better!